src/index.js

```diff
@@ -1,59 +1,57 @@
-import rpcServer from "./rpc-server";
+import rpcServer from './rpc-server';
 import {getRpcClient, initRpcClient} from './rpc-client';
-import { getLogger, setLogger } from "./logger";
+import {getLogger, setLogger} from './logger';
 
-initRpcClient((jsonRPCRequest) => {
+initRpcClient(jsonRPCRequest => {
   postMessage({
-    type: "json-rpc-request",
+    type: 'json-rpc-request',
     body: jsonRPCRequest,
   });
 
   return Promise.resolve(jsonRPCRequest);
 });
 
-const postMessage = (message) =>
+const postMessage = message =>
   window.ReactNativeWebView
     ? window.ReactNativeWebView.postMessage(JSON.stringify(message))
-    : console.log("POST Message", message);
+    : console.log('POST Message', message);
```

Comment:
what's the console log for? Debugging? Should it use winston logging?

Reply:
It was used just when running unit tests. But i've removed already, its not required.